### PR TITLE
- minor adjustements for change notifications AEH delivery mode

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: concepts/change-notifications-delivery.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ ms.custom: graphiamtop20
 
 Change notifications can be delivered in different ways to subscribers. If the main delivery mode for change notifications is through webhooks, it can be challenging to take advantage of webhooks for high throughput scenarios or when the receiver cannot expose a publicly available notificiation URL.  
 
+As the change notifications delivery mode is a platform-board capability, it is available for all resources that support Microsoft Graph change notifications.
```
Comment (davidmu1) [inline concepts/change-notifications-delivery.md:13]: I'm not sure what "platform-board" means. Will customers?

Suggested change:

Change notifications are available for all resources that support Microsoft Graph change notifications.

File: concepts/change-notifications-delivery.md [hunk 2]
```diff
@@ -18,7 +20,7 @@ Good examples of high throughput scenarios include applications subscribing to a
 Using Azure Event Hubs to receive change notifications differs from webhooks in a few ways, including:
 
 - You don't rely on publicly exposed notification URLs. The Event Hubs SDK will relay the notifications to your application.
-- You don't need to implement the [notification URL validation](webhooks.md#notification-endpoint-validation).
+- You don't need to reply to the [notification URL validation](webhooks.md#notification-endpoint-validation). You will receive a validation message that you need to discard.
```
Comment (davidmu1) [inline concepts/change-notifications-delivery.md:23]:
```suggestion
- You don't need to reply to the [notification URL validation](webhooks.md#notification-endpoint-validation). You can ignore the validation message that you receive.
```

File: concepts/change-notifications-delivery.md [hunk 3]
```diff
@@ -136,6 +138,27 @@ Before you can receive the notifications in your application, you'll need to cre
 
 > **Note:** Create a separate policy for the application that listens to Event Hubs messages instead of reusing the same connection string you set in Azure KeyVault. This ensures that each component of the solution has only the permissions it needs and follows the least permissions security principle.
 
+> **Note:** Your application will receive validation messages whenever it creates a new subscription, you should discard this type of notifications. The example below represents the body of a validation message.
```
Comment (davidmu1) [inline concepts/change-notifications-delivery.md:141]:
```suggestion
> **Note:** Your application receives validation messages whenever it creates a new subscription, you should ignore these notifications. The example below represents the body of a validation message.
```

File: concepts/change-notifications-delivery.md (hunk 1 above)
Comment (baywet) [reply concepts/change-notifications-delivery.md:13]:
```suggestion
This change notifications delivery mode is available for all resources that support Microsoft Graph change notifications.
```

**Pull request merged**